dev/docs/styles.html

```diff
@@ -4,7 +4,7 @@
 @import 'https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Sharp|Material+Icons+Round';
 </style>
 
-<link href="https://flutter.github.io/cupertino_icons/css/icons.css" rel="stylesheet" type="text/css">
+<link href="https://flutter.github.io/assets-for-api-docs/assets/cupertino/assets/cupertino.css" rel="stylesheet" type="text/css">
```

Comment:
```suggestion
<link href="https://flutter.github.io/assets-for-api-docs/assets/cupertino/cupertino.css" rel="stylesheet" type="text/css">
```

This url is slightly different from the one actually deployed.

Reply:
Ah, I see. It seems that I checked the code at a too early timing. 😅

Reply:
You saved me a few minutes of head-scratching staring at that URL, I appreciate it!